modules/ocf/manifests/repackage.pp

```diff
@@ -1,37 +1,29 @@
 define ocf::repackage(
     $package     = $title,
-    $recommends  = true,
+    $recommends  = false,
```

Comment:
I think this should still default to true, as is the Debian default.

With this change, it's not obvious why we do `ocf::repackage` in some places instead of just `package` to someone looking at the code. With `ocf::repackage { 'package': recommends => false }` it's immediately obvious we're doing it to not install recommends.

Plus, it's weird that we have to set `recommends` back to `true` for packages we want to backport. `ocf::repackage` does multiple things, but it probably shouldn't do any of them by default.